heuristic/heuristic_calculations.py

```diff
@@ -33,7 +33,9 @@ def calculate_weekly_rest(data, x, w):
 
             if actual_shifts[key][0][0] - important[week] >= 36:
                 off_shift_periods[key].append((important[week], actual_shifts[key][0][0] - important[week]))
-
+                
+    for key in off_shift_periods:
+            w[key] = max(off_shift_periods[key],key=itemgetter(1))
```

Comment:
Use 
```python
min(
    WEEKLY_REST_HOURS[1],
    max(off_shift_periods[key],key=itemgetter(1))
```

here?

Reply:
I think I will continue to use 
'''
w[key] = max(off_shift_periods[key],key=itemgetter(1))
'''
It makes more sense to actually give assign the employee to the rest period even though they are not rewarded for it. Or else we would just cut off rest periods and that would seem weird in the visualisation of the schedules. The restriction on reward should be maintained in the objective value calculations

Reply:
Fair point!